docs/conf.py

```diff
@@ -208,4 +209,16 @@
 # -- Options for todo extension ----------------------------------------------
 
 # If true, `todo` and `todoList` produce output, else they produce nothing.
-todo_include_todos = True
+# todo_include_todos = True
+
+# -- Nbsphinx settings --------------------------------------------------------
+nbsphinx_allow_errors = True
```

Comment:
I think it might be better for it to fail upon an error, as otherwise our online documentation examples could contain errors.  For example notebook titled "Isotropic Rotational Diffusion model convoluted with resolution function and fitted with bumps" fails to import some modules and so is largely cells which have raised errors rather than executing correctly.

Reply:
I set nbsphinx_allow_errors to False